### DISCUSSION TITLE
[Feature Request] Figma file for NextUI components

### DISCUSSION THREAD

**Discussion by tnv1r** [heroui-inc/heroui#552] [Feature requests]
### Is your feature request related to a problem? Please describe.

Building a Figma design that corresponds & syncs with the actual code implementation is really hard. It really takes time to make designs with a component library in mind.

### Describe the solution you'd like

 I feel like NextUI already has a design system, would love it, if they make it public for the community. 

### Describe alternatives you've considered

There are libraries like ChakraUI which have their own Figma component library open source. Flowbite and Untitled-UI also made their component library public. (even tho they have a premium version). The least NextUI can do is, they can have a community-free version to get started with the design.

**Comment by ShellarFX** [comment]
Figma file will be published with NextUI release

**Comment by smorimoto** [comment]
Is there a candidate date for the next release?

**Comment by GeoffCreation** [comment]
Any update on NextUI Figma component library release?

**Comment by AmauryAparicio** [comment]
Where can I download the Figma file?

**Comment by salvinoto** [comment]
Any idea when the figma will be released?

**Comment by GBDBEG1** [comment]
Issue #516 stated a figma file was coming in next release, but 1.0.0 doesn't seem to include one? Was there one released and we just need a PR to update the docs to point to it?

**Reply by ShellarFX** [reply]
Release of stable version, not beta. For now it is still beta

**Comment by BlockifyTechnologies** [comment]
Any updates on this ?

**Reply by upfounders** [reply]
Any update on the release date of Figma file ?

**Comment by jrgarciadev** [comment]
https://www.figma.com/community/file/1267584376234254760
